src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -6,12 +6,111 @@
 #include "winrt/Microsoft.UI.Xaml.Controls.h"
 
 #include "TerminalPage.g.h"
+#include "Tab.h"
+#include "CascadiaSettings.h"
+#include "Profile.h"
+#include "ScopedResourceLoader.h"
+
+#include <winrt/Microsoft.Terminal.TerminalControl.h>
+#include <winrt/Microsoft.Terminal.TerminalConnection.h>
+#include <winrt/Microsoft.UI.Xaml.Controls.Primitives.h>
+#include <winrt/Windows.ApplicationModel.DataTransfer.h>
+#include <winrt/Microsoft.UI.Xaml.XamlTypeInfo.h>
 
 namespace winrt::TerminalApp::implementation
 {
     struct TerminalPage : TerminalPageT<TerminalPage>
     {
+    public:
         TerminalPage();
+
+        void SetSettings(::TerminalApp::CascadiaSettings* settings, bool needRefreshUI);
+
+        void Create();
+
+        hstring GetTitle();
```

Comment:
nit: to be like other C++/WinRT properties, this should just be `hstring Title() const`.

Reply:
changed